### PR TITLE
FWF507: fix sort outsideclick

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
Issue Description: Unable to remove the list on clicking on filter option ""Created"

UI screen:

image

### CODE REVIEW

File: camunda-formio-tasklist-vue/src/components/layout/Header.vue [hunk 1]
```diff
@@ -23,45 +23,48 @@
       </div>
       <FormListModal :token="token" :bpmApiUrl="bpmApiUrl"/>
       <div class="cft-first">
-        <!-- Sorting section -->
 				<div id="cftf-dpdown-container" class="mx-2">
 					<div class="cftf-dpdown-box mr-2" v-for="(sort, idx) in sortList" :key="sort.sortBy">
```
Comment (ann-aot) [inline camunda-formio-tasklist-vue/src/components/layout/Header.vue:27]: pls correct the spacing

**Review by ann-aot — APPROVED** [review]

**Comment by ann-aot** [comment]
there are some conflicts- pls resolve the conflicts

**Comment by kurianbenoy-aot** [comment]
@ann-aot please don't merge this PR for the time being

File: camunda-formio-tasklist-vue/src/components/layout/Header.vue (hunk 1 above)
Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/layout/Header.vue]: fixed the spacing issue

**Comment by kurianbenoy-aot** [comment]
I have resolved the conflict and fixed the issues after merging. Now the outside click is working fine

image

**Comment by kurianbenoy-aot** [comment]
Merging #7

**Pull request merged**